html/semantics/scripting-1/the-script-element/resources/set-window-executed.js

```diff
@@ -0,0 +1 @@
+window.executed = true;
```

Comment:
Maybe this could set `document.currentScript.executed = true` instead, to avoid keeping track of ordering in tests?